mmv1/templates/terraform/examples/pubsub_subscription_push_cloudstorage.tf.erb

```diff
@@ -0,0 +1,42 @@
+resource "google_storage_bucket" "<%= ctx[:primary_resource_id] %>" {
+  name  = "<%= ctx[:vars]['bucket_name'] %>"
+  location = "us-central1"
```

Comment:
looking through other similar tests that grant SA permissions on the bucket, many seem to have 
```
  uniform_bucket_level_access = true
```
enabled on the bucket. Looking through https://cloud.google.com/storage/docs/access-control/iam-roles#basic-roles-modifiable it seems to have quite a few behaviors. Maybe that could help the test consistency?

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/datastream_stream_basic.tf.erb seems to do that, along with a "viewer" role. That test currently passes

Otherwise, we could try enabling `storage.admin` instead.

Reply:
I added the the suggested field to enable uniform bucket access.